bin/templates/lib/yarp/serialize.rb.erb

```diff
@@ -12,9 +12,11 @@ module YARP
       attr_reader :source, :io, :encoding
 
       def initialize(source, io)
-        @source = source.b
-        @io = io
+        # This is wrong, we should be dumping and loading the encoding. For now,
```

Comment:
Does this project have a convention of using FIXME: for things like this?

Reply:
In case it's not clear, it was an existing comment, I just moved it to a better place.
I can prepend it with FIXME/TODO.

Reply:
Ah this is already tracked as https://github.com/ruby/yarp/issues/643